app/src/main/java/org/p2p/wallet/swap/jupiter/api/SwapJupiterApi.kt

```diff
@@ -18,9 +17,9 @@ interface SwapJupiterApi {
     ): CreateSwapTransactionResponse
 
     @GET("v4/quote")
-    suspend fun getQuote(
-        @Query("inputMint") inputMint: Base64String,
-        @Query("outputMint") outputMint: Base64String,
+    suspend fun getSwapRoutes(
+        @Query("inputMint") inputMint: Base58String,
```

Comment:
have checked that retrofit takes base58Value inside instead of calling Base58String.toString()?

Reply:
org.p2p.wallet.home.model.Base58TypeAdapter
yes